src/main/java/dev/meyi/bn/commands/BazaarNotifierCommand.java

```diff
@@ -1,24 +1,29 @@
 package dev.meyi.bn.commands;
 
 import dev.meyi.bn.BazaarNotifier;
+import dev.meyi.bn.utilities.EnchantedCraftingHandler;
 import dev.meyi.bn.utilities.Defaults;
+import dev.meyi.bn.utilities.Suggester;
 import dev.meyi.bn.utilities.Utils;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.List;
+import java.util.*;
+
 import net.minecraft.command.CommandBase;
 import net.minecraft.command.ICommandSender;
 import net.minecraft.entity.player.EntityPlayer;
 import net.minecraft.event.ClickEvent;
 import net.minecraft.event.ClickEvent.Action;
+import net.minecraft.util.BlockPos;
```

Comment:
Is this import necessary? We shouldn't be using this in the command (or anywhere in the mod for that matter).

Reply:
I see what the problem was.